CMakeLists.txt

```diff
@@ -168,6 +168,11 @@ if(ENABLE_PYTHON)
     target_compile_definitions(k40gen PRIVATE "-DXTENSOR_ENABLE_XSIMD")
   endif()
 
+  # xtensor headers
+  find_package(xtensor REQUIRED)
```

Comment:
Does it also work without this line?

Reply:
yes, it works without that line, I'll update it